scripts/build/nightly_build.sh

```diff
@@ -20,11 +20,15 @@ while fuser /var/lib/dpkg/lock >/dev/null 2>&1 ; do
   echo "Waiting for apt-get to finish"
 done
 
+# Download additional dependencies
+cd ${HOME}
+wget https://github.com/loumalouomega/MMGPrecompiled/archive/master.zip -O mmg.zip
```

Comment:
wait, wouldn't it make more sense to add the libs to the KratosMultiphysics git account rather than yours?
There we could also put Metis, Trilinos, ...
@roigcarlo is this possible?

Reply:
@roigcarlo told me to do it this way

Reply:
I was not specifically implying that it had to be in @loumalouomega account. Personally I don't want to turn kratos into a repo of compiled software to avoid problems (mainly because then we can be considered as software distributors). Of course sames goes for your @loumalouomega's github account.

My initial thoughts were more in the lines of a FTP server or something like that. Sorry If I did not explain myself clearly. Seeing that this is a problem what the others members of the @KratosMultiphysics/technical-committee think?

Reply:
Guys, i think we should get in contact with people at MMG to ask (in writing) if they agree about us doing this... you have the contact of Algiane, Heloise and Mario, why don't you try to write them and ask?

Reply:
I added a direct link, in any case there is no problem to  have a precompiled version of a LGPL program